packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -4,35 +4,44 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-
+import {
```

Comment:
My gut feeling tells me that this file has too many lines of comments. 
I worry that these are going to go out of date and no one will maintain them.
We should evaluate this and see if any of the comments should be removed in lieu of extracting things into new methods with better names.

Reply:
Removed.